playroom/components.tsx

```diff
@@ -130,11 +130,11 @@ const PreviewToolsControls: React.FC<PreviewToolsControlsProps> = ({
 }) => {
     const classes = useControlsStyles();
     const {colors} = useTheme();
-    const {isMobile} = useScreenSize();
+    const {isTabletOrSmaller} = useScreenSize();
```

Comment:
Can you force to show Preview tools as Tab instead of Select form in tablet breakpoint?

Reply:
sure